mlrun/projects/project.py

```diff
@@ -2424,9 +2424,18 @@ def _instantiate_function(
             )
 
         if isinstance(func, str):
-            # in hub or db functions name defaults to the function name
-            if not name and not (func.startswith("db://") or func.startswith("hub://")):
-                raise ValueError("Function name must be specified")
```

Comment:
What about raising this error: `raise ValueError("Function name must be specified")`